demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -573,10 +579,14 @@ def validate_script(self, structure_validator, pack_error_ignore_list, is_modifi
             return deprecated_result
 
         if is_modified and self.is_backward_check:
-            return all([script_validator.is_valid_file(validate_rn=False),
+            return all([script_validator.is_valid_file(validate_rn=False,
+                                                       check_unskipped_playbooks=self.check_unskipped_playbooks,
+                                                       id_set_file=self.id_set_file),
                         script_validator.is_backward_compatible()])
         else:
-            return script_validator.is_valid_file(validate_rn=False)
+            return script_validator.is_valid_file(validate_rn=False,
+                                                  check_unskipped_playbooks=self.check_unskipped_playbooks,
+                                                  id_set_file=self.id_set_file)
```

Comment:
a thought - since we are using the id_set here this should be checked with the id_set validator - you can find the call to it in `validate_manager.py` in line 382.
![image](https://user-images.githubusercontent.com/50295826/120930295-575e0a00-c6f5-11eb-9884-6f8705d8fb7e.png)

That way we do not need to add anything to the is_valid_file() method - true for integrations as well.